pkg/guestagent/guestagent_linux.go

```diff
@@ -324,6 +324,11 @@ const deltaLimit = 2 * time.Second
 
 func (a *agent) fixSystemTimeSkew() {
 	for {
+		ok, err := timesync.HasRTC()
+		if !ok {
+			logrus.Warnf("fixSystemTimeSkew: error: %s", err.Error())
```

Comment:
You can use logrus.WithError

Reply:
ping @afbjorklund

Reply:
I can add it, just didn't know if it was required (the other code is not using it yet)